README.md

```diff
@@ -168,6 +168,33 @@ Dato::Client.new.items.update(attributes: { title: 'Hello world' }, item_id: '12
 Dato::Client.new.items.destroy(item_id: '123')
 ```
 
+## File upload
+
+Dato Rails also supports file uploads. 
+These can be created either from a local file or from a url. 
+Basically all file types are supported, as long as they are valid in the CMS.
+
+### Upload from Url
+
+```ruby
+  Dato::Client.new.uploads.create_from_url(url: 'https://picsum.photos/seed/picsum/200/300https://picsum.photos/seed/picsum/200/300')
```

Comment:
```suggestion
  Dato::Client.new.uploads.create_from_url('https://picsum.photos/seed/picsum/200/300https://picsum.photos/seed/picsum/200/300')
```

I'd prefer it like this because it's only one parameter and it's clear from the method name that is an url

Reply:
Thanks for the feedback, makes sense to remove the named parameter. And what about create_from_file?